### PR TITLE
[xargs] Update docblock comment

### PR DESCRIPTION
**Pull request by ryanrhee**

### CODE REVIEW

File: pre_commit/xargs.py [hunk 1]
```diff
@@ -106,6 +106,7 @@ def _thread_mapper(maxsize):
 def xargs(cmd, varargs, **kwargs):
     """A simplified implementation of xargs.
 
+    color: Make a TTY
```
Comment (asottile) [inline pre_commit/xargs.py]: technically it's a `pty`

Comment (asottile) [reply pre_commit/xargs.py]: (and only on platforms which support it)

Comment (ryanrhee) [reply pre_commit/xargs.py]: tru

Comment (ryanrhee) [inline pre_commit/xargs.py]:
```suggestion
    color: Make a pty
```

File: pre_commit/xargs.py [hunk 2]
```diff
@@ -106,6 +106,7 @@ def _thread_mapper(maxsize):
 def xargs(cmd, varargs, **kwargs):
     """A simplified implementation of xargs.
 
+    color: Make a pty
```
Comment (ryanrhee) [inline pre_commit/xargs.py]:
```suggestion
    color: Make a pty on platforms that support it
```

File: pre_commit/xargs.py (hunk 1 above)
Comment (ryanrhee) [reply pre_commit/xargs.py]: updated

**Pull request merged**